### DISCUSSION TITLE
【V5】如何实现 V4 annotation line（一般情况）

### DISCUSSION THREAD

**Discussion by BinghuiXie** [antvis/G2#5371] [Q&A]
- **G2 Version**: 5.0.16
- **Platform**:
- **Mini Showcase(like screenshots)**:
- **CodePen Link**:

![image](https://github.com/antvis/G2/assets/34343589/5ed5cd16-233d-4481-b3a0-33e0fb5fbeef)
V4 可以通过 `start` 和 `end` 自定义 `annotation` 线的起始点和终点，但是在 V5，目前只给出了水平和垂直两种类似 `annotation` `line` 的实现，更一般的情况，在 V5 中如何实现
PS：V4 中的 `annotation` 不会影响已有的坐标系的范围，同样效果如何在 V5 中实现？

**Comment by ai-qing-hai** [comment]
@BinghuiXie   使用 link 尝试一下。
![image](https://github.com/antvis/G2/assets/65594180/2912cfc7-5e5c-424d-8c46-956260ab3f24)
